yb-voyager/cmd/importData.go

```diff
@@ -972,6 +972,14 @@ func executeSqlStmtWithRetries(conn **pgx.Conn, sqlInfo sqlInfo, objType string)
 			time.Sleep(time.Second * 5)
 			log.Infof("RETRYING DDL: %q", sqlInfo.stmt)
 		}
+
+		if bool(flagPostImportData) && strings.Contains(objType, "INDEX") {
+			//In case of index creation print the index name as index creation takes time
+			//and user can see the progress
+			if sqlInfo.objName != "" {
+				color.Yellow("creating index %s ...", sqlInfo.objName)
```

Comment:
This is a step in the right direction. Whenever it is possible, we should add two more pieces of information here:
- total number of rows in the corresponding table
- after the index creation completes, the amount of time it took to create the index.

These two pieces of information gives an approximate idea to the user about how much time it takes to create an index relative to its size.

Reply:
Yeah, it makes sense. I will add an enhancement ticket for the future. we can see when to pick it.

Reply:
https://yugabyte.atlassian.net/browse/DB-9632